services/provider/server/server.go

```diff
@@ -769,3 +765,43 @@ func (s *OCSProviderServer) getOCSSubscriptionChannel(ctx context.Context) (stri
 	}
 	return subscription.Spec.Channel, nil
 }
+
+func (s *OCSProviderServer) getClusterResourceQuotaSpec(consumerResource *ocsv1alpha1.StorageConsumer) quotav1.ClusterResourceQuotaSpec {
```

Comment:
There is no reason for this function to be attached to the `ProviderServer` object. Please make it a stateless standalone function.